content/php/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'round()'
+Description: 'Round a number to the nearest whole number or to specified decimal numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`round()`** converts an integer or a float to the nearest whole number or to specified decimal numbers.
+
+## Syntax
+
+```pseudo
+$roundedNumber = round($number);
```

Comment:
```suggestion
$roundedNumber = round($number, $decimals, $roundMode);
```

Reply:
I recommend having one syntax block with all the parameters. And address each parameter below.